src/components/Table/Table.tsx

```diff
@@ -27,76 +30,90 @@ export type Column = {
 };
 
 export interface TableProps extends DefaultProps {
+	bordered?: boolean;
+	children?: ReactNode;
+	columns: Column[];
 	data: any[];
+	emptyStateMessage?: string;
+	horizontal?: boolean;
+	onColumnClick?: (id: string) => void;
+	renderCell?: (row: any, cell: any, rowIndex: number, cellIndex: number) => ReactNode;
 	rowKey: string;
-	columns: Column[];
 	sortColumn?: string;
 	sortOrder?: 'asc' | 'desc';
 	styled?: boolean;
-	bordered?: boolean;
-	emptyStateMessage?: string;
-	renderCell?: (row: any, cell: any, rowIndex: number, cellIndex: number) => ReactNode;
-	onColumnClick?: (id: string) => void;
+	untable?: boolean;
 }
 
 export const Table: FunctionComponent<TableProps> = ({
+	bordered,
+	children,
 	className,
+	columns = [],
 	data = [],
+	emptyStateMessage,
+	horizontal,
+	onColumnClick = () => {},
+	renderCell = () => null,
 	rowKey,
-	columns = [],
 	sortColumn,
 	sortOrder = 'asc',
 	styled,
-	bordered,
-	emptyStateMessage,
-	renderCell = () => null,
-	onColumnClick = () => {},
+	untable,
 }) => {
 	return (
-		<Fragment>
+		<>
```

Comment:
Overal in de code gebruiken we Fragment. Mss beter dat we consequent blijven? Of heeft `<>` een andere betekenis dan `<Fragment>`?